src/components/create-project-form/inputs/ProjectImageInput.js

```diff
@@ -31,6 +36,20 @@ export const ProjectImageInput = ({
       setDisplayImage(await toBase64(acceptedFile[0]))
     }
   })
+
+  const ProjectImage = type => {
+    return (
+      <div
+        style={{
+          width: '100%',
+          height: '100%',
+          backgroundSize: 'cover',
+          backgroundImage: `url('/assets/create/projectImageGallery${type.toString()}.svg')`
```

Comment:
Do we save all images - also pixel images as svg's? As far as I understand we at least limit file type in useDropzone to image ... could that also include not compatible images?

This could lead to crazy stuff if we dont cap resolutions and filesize - should we think about a proper ingestion module (there are tons out in the wild)?

Reply:
I agree. I'm not up to date on the image handling as it was work made by @aminlatifi and @brymut. Also, I want to make sure what's going on with the edition, are we removing the previous data or just changing it